source/collections/VectorList.ooc

```diff
@@ -74,7 +74,7 @@ VectorList: class <T> {
 	}
 	pop: func -> T {
 		result := this[0]
-		this remove(0)
+		this removeAt(0)
```

Comment:
Do you mean 
```ooc
pop: func -> T {
    this remove(0)
}
```
or
```ooc
pop: func -> T {
    result := this remove(0)
    result
}
```
would work, too, and not leak memory? Then why aren't we doing that instead?